src/core/Akka.TestKit/TestKitBase_ExpectAllWithPredicates.cs

```diff
@@ -58,7 +58,7 @@ public IReadOnlyCollection<object> ExpectMsgAllOfMatchingPredicates(TimeSpan max
     {
         max.EnsureIsPositiveFinite("max");
         var enumerable = InternalExpectMsgAllOfAsync(Dilated(max), predicates, cancellationToken: cancellationToken)
-            .ConfigureAwait(false).WithCancellation(cancellationToken);
+            .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.